rclcpp/include/rclcpp/node.hpp

```diff
@@ -127,7 +128,9 @@ class Node
     size_t queue_size,
     std::function<void(const std::shared_ptr<MessageT> &)> callback,
     rclcpp::callback_group::CallbackGroup::SharedPtr group = nullptr,
-    bool ignore_local_publications = false);
+    bool ignore_local_publications = false,
+    typename rclcpp::msg_memory_strategy::MessageMemoryStrategy<MessageT>::SharedPtr
+    msg_mem_strat = rclcpp::msg_memory_strategy::MessageMemoryStrategy<MessageT>::create_default());
```

Comment:
uncrustify did this for me, I originally indented 133 by one more level. I think it will always try to put continued lines on the same indentation level, even if one of the arguments wraps two lines. I can use the INDENT-OFF annotation unless you come up with a better way.